lib/config.js

```diff
@@ -37,7 +35,8 @@ export default class Config {
         jade: Joi.string().default('**/*.jade'),
         css: Joi.string().default('**/*.+(css|sss)'),
         js: Joi.string().default('**/*.js'),
-        static: Joi.string().default(`**/*.{${union(binaryExtensions, imageExtensions).join(',')}}`)
+        // this has to not match any webpack entry file
+        static: Joi.string().default('!**/*.+(js|css|sss|jade)')
```

Comment:
hmm good question. I'm not sure, that seems very heavy-handed. We might not always want to use sugarss, and surely some other people wouldn't either. I am comfortable forcing postcss, but I'm not as comfortable forcing one particular syntax...